lib/api/authn/webid-oidc.js

```diff
@@ -23,7 +23,7 @@ const {
  * @param app {Object} Express.js app instance
  * @param argv {Object} Config options hashmap
  */
-function initialize (app, argv) {
+function initialize(app, argv) {
```

Comment:
Please remove minor changes, such as removing spaces, as they make merging more difficult; We'll have a cleanup-issue at some point to make sure that all files follow a certain format. (We already have linting, but it's not very strict on this.)

Reply:
@megoth which linter are you using Jslint/eslint? If not ,can you please create another feature to implement eslint in this code repository,and assign it to me?